core/src/main/java/org/apache/sdap/mudrod/discoveryengine/WeblogDiscoveryEngine.java

```diff
@@ -105,7 +105,7 @@ public void preprocess() {
       startTime = System.currentTimeMillis();
       LOG.info("Processing logs dated {}", anInputList);
 
-      DiscoveryStepAbstract im = new ImportLogFile(this.props, this.es, this.spark);
+     /* DiscoveryStepAbstract im = new ImportLogFile(this.props, this.es, this.spark);
```

Comment:
OK, the issue SDAP-161 is described as being **MUDROD embedded unit test** there should therefore be no code other than en embedded Unit test...

Reply:
Additionally, if you are going to comment out code that you do not intend to use... just remove it.